<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#9BA4B4;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">color: white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>校验方式</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbt_Check_XOR">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>异或校验</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbt_Check_LRC">
        <property name="text">
         <string>LRC 校验</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="rbt_Check_Sum">
        <property name="text">
         <string>CheckSum 校验</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="tbtn_help">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/img/imgs/help.png</normaloff>:/img/imgs/help.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>需要校验的数据</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbt_HEX">
        <property name="text">
         <string>HEX</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextEdit" name="te_data"/>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbt_ASCII">
        <property name="text">
         <string>ASCII</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">color: white;
font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>校验结果</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="2">
       <widget class="QToolButton" name="tbtn_OCT">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/img/imgs/copy.png</normaloff>:/img/imgs/copy.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>开始校验</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: white;
font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>BIN：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: white;
font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>HEX：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: white;
font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>OCT：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="tbtn_BIN">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/img/imgs/copy.png</normaloff>:/img/imgs/copy.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tbtn_HEX">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/img/imgs/copy.png</normaloff>:/img/imgs/copy.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="tbtn_DEC">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/img/imgs/copy.png</normaloff>:/img/imgs/copy.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: white;
font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>DEC：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="te_HEX"/>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="te_DEC"/>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="te_OCT"/>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="te_BIN"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
